### PR TITLE
Pubsub

### PR DESCRIPTION
**Pull request by arroyoAle**
# What and why?
Setting up PubSub connection to complete survey
# How to test?

# Trello

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ structlog = "*"
 gunicorn = "*"
 gevent = "*"
 sdc-cryptography = "*"
+google-cloud-pubsub = "==1.7.0"
```
Comment (insacuri) [inline Pipfile:12]: Can you try and use * instead.  For most of our repos we do and it works so it should *hopefully* work here

File: mock_eq/routes.py [hunk 2]
```diff
@@ -1,41 +1,51 @@
...
 import json
 
-import logging
+logger = structlog.wrap_logger(logging.getLogger(__name__))
 
-KEY_PURPOSE = "authentication"
-log = logging.getLogger()
-logging.basicConfig()
+_health_check = {}
+if Path("git_info").exists():
```
Comment (insacuri) [inline mock_eq/routes.py:15]: Can you get rid of this git_info stuff?  It's always been fairly pointless in our other repos so no need to duplicate it here

File: git_info [hunk 3]
```diff
@@ -0,0 +1 @@
+{"test": "test"}
```
Comment (insacuri) [inline git_info:1]: Can you delete this file?  It's not really useful in other repos so no need to duplicate it here.

File: mock_eq/templates/layouts/_base.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  "header": {
+    "title": "Mock eQ",
+  }
+} %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+{%- block pageContent %}
+  <div class="page__container container">
+   {% block breadcrumb %}{% endblock breadcrumb %}
+    {% if availability_message %}
```
Comment (insacuri) [inline mock_eq/templates/layouts/_base.html:27]: You can remove this availability message stuff (Lines 27 to 34).  That's for something specific to frontstage that isn't in this repo

File: _infra/helm/mock-eq/templates/deployment.yaml [hunk 5]
```diff
@@ -21,36 +25,23 @@ spec:
...
-            cpu: "100m"
-            memory: "100Mi"
+          {{ toYaml .Values.resources | nindent 12 }}
         env:
         - name: JSON_SECRET_KEYS
           valueFrom:
             secretKeyRef:
               name: mock-eq
               key: mock-eq
-      # readinessProbe:
```
Comment (insacuri) [inline _infra/helm/mock-eq/templates/deployment.yaml:40]: I think this stuff is actually needed to let k8s know how to ask the app if it's healthy/ready.  Can you add this stuff back?  In theory it should just work as it's very similar to others

Can you also add a very simple /info endpoint to this repo.  Just something that returns something like:
```python
{"status": "healthy"}
```
or something basic like that.

**Review by insacuri — CHANGES_REQUESTED** [review]
A few small changes but it's looking good.  I'll deploy it to my environment tomorrow to play with it a bit, but it shouldn't take too much more effort to get this in!

File: mock_eq/routes.py [hunk 6]
```diff
@@ -1,41 +1,54 @@
...
+    return redirect(app.config["FRONTSTAGE_URL"])
+
+
+@app.route('/info', methods=["GET"])
+def info():
+    info = {
+        "name": "mock-eq",
+        "status": "healthy",
+    }
+    return make_response(jsonify(info), 200)
```
Comment (insacuri) [inline mock_eq/routes.py:54]: This can be simplified to 
```python
return jsonify(info)
```

File: config.py [hunk 7]
```diff
@@ -2,6 +2,9 @@
 
 
 class DevelopmentConfig(object):
+    VERSION = "1.0.9"
```
Comment (insacuri) [inline config.py:5]: This isn't needed.  I don't think anything uses this

**Review by insacuri — APPROVED** [review]

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**